### PR TITLE
Adding Integrations User Guide from AKN

### PR DESCRIPTION
**Pull request by drewavisautodesk**
Trying again... adding the content from the Integrations User Guide on AKN to the developer help.

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/01eada2acc1f00bf670fb8fc721f9e3a7c376a73/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/01eada2acc1f00bf670fb8fc721f9e3a7c376a73/index.html)

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+pagename: integrations-user-guide
+lang: en
+---
+
+<!-- omit in toc -->
+# Integrations user guide
+
+In this guide:
+- [Introduction](#introduction)
+- [Getting started with {% include product %} Desktop](#getting-started-with--include-product--desktop)
```
Comment (shaynacohen) [inline docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md:13]: This link isn't working, maybe try it without the extra dashes?

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+  - [Installation of Desktop](#installation-of-desktop)
+  - [Logging in](#logging-in)
+    - [Authentication and browser integration](#authentication-and-browser-integration)
+  - [The system tray](#the-system-tray)
+  - [The project browser](#the-project-browser)
+  - [The command launcher](#the-command-launcher)
+  - [The console](#the-console)
+- [Launching applications](#launching-applications)
+- [The {% include product %} Panel](#the--include-product--panel)
```
Comment (shaynacohen) [inline docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md:22]: This link is having the same issue, maybe it's the variable or the extra dashes?

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md [hunk 3]
```diff
@@ -0,0 +1,509 @@
...
+
+You will only need to update your security certificate list once. Please note that only Mac and Windows show a prompt for certificates, not Linux.
+
+Once that's set up, you'll need to log in to your {% include product %} site.
+
+If you go through a proxy, you will have to do some additional configuration. See instructions on how to do that in our [Admin Guide](https://developer.shotgridsoftware.com/8085533c/?title=ShotGrid+Integrations+Admin+Guide#toolkit-configuration-file) .
+
+#### Authentication and browser integration
+
+Desktop provides services that allow the {% include product %} web application to interact with your computer. This enables features like [local file linking](topicid=SG_Administrator_ar_data_management_ar_linking_local_files_html) and software launching. For security reasons, you must be logged into Desktop as the same user on the same {% include product %} site as you are in the browser. If you are logged in as a different user or on a different site, Desktop will prompt you to switch accounts and if you do it will restart and prompt for the appropriate credentials.
```
Comment (shaynacohen) [inline docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md:100]: Update this link to: https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_data_management_ar_linking_local_files_html

**Review by shaynacohen — COMMENTED** [review]
@drewavisautodesk - I left a couple of more minor fix notes, but it looks great! I'll leave it up to you if you want to fix before merging, or fix after the merge.

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md (hunk 1 above)
Comment (drewavisautodesk) [reply docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md:13]: I think it's the include variable inside the link, I'll see if I can replace that with the variable

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/9c0b73352699cf903f9231cb942caf3f3724fb05/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/43a389dc1b08f0af9cb7ff3dbe8067fa3c86c665/index.html)

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md (hunk 1 above)
Comment (drewavisautodesk) [reply docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md]: I've had to remove the product name variable completely from the title in order to fix this issue.

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md (hunk 2 above)
Comment (drewavisautodesk) [reply docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md]: Some solution here.

File: docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md (hunk 3 above)
Comment (drewavisautodesk) [reply docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md]: Link updated

**Review by shaynacohen — APPROVED** [review]

**Comment by shaynacohen** [comment]
LGTM!

**Pull request merged**